code/datums/status_effects/gas.dm

```diff
@@ -5,6 +5,7 @@
 	alert_type = /obj/screen/alert/status_effect/freon
 	var/icon/cube
 	var/can_melt = TRUE
+	var/icewing = FALSE //Is checked to prevent icewing watcher freezing blasts from warming you up when they expire
```

Comment:
uhh dont you only get frozen solid from actual icewing watchers

Reply:
Yes, and that variable is only checked in the proc when you're frozen by that status effect. I didn't want to change how any other freezing mechanic worked that I didn't know about, because of things like the wizard's freezing rune exists. 
I figured I'd leave everything tied to that proc working exactly as it was before, and I'd have icewing watchers skip out on the line that restores your body temperature to normal.
If I didn't do that then icewing watchers would be **less** of a threat to lizard people than normal watchers, because they'd get chilled and frozen, then once the freeze expired they'd no longer be chilled, which is just silly.